src/main/java/seedu/address/logic/parser/ViewAvailableCommandParser.java

```diff
@@ -76,10 +67,37 @@ public ViewAvailableCommand parse(String args) throws ParseException {
                         + ViewAvailableCommand.MESSAGE_USAGE);
             }
 
+            if (isPastDate(date)) {
+                throw new ParseException(MESSAGE_PAST_DATE + "\n" + "Please insert in the following format: " + "\n"
+                        + ViewAvailableCommand.MESSAGE_USAGE);
```

Comment:
error thrown can just be MESSAGE_PAST_DATE with a bit more details

e.g. Date entered has past already! Date must be after current date 

should be sufficient, instead of including message usage. Also, could you de-capitalise DATE to just Date

follows as per other errors thrown

Reply:
Thanks for the suggestion

DATE is fully capitalized because it is a parameter so I think it is fine to leave it fully capped

I think the format is fine also, could be good to let the user know the correct command after he inserts something wrong as seen
![image](https://github.com/AY2324S1-CS2103T-W09-3/tp/assets/122218312/884b3d7f-fbcf-4f85-92f5-fb9b95bd9775)